build_tools/cmake/build_and_test_asan.sh

```diff
@@ -130,6 +130,7 @@ declare -a excluded_tests=(
   "iree/tests/e2e/models/mnist_fake_weights.mlir.test"
   "iree/tests/e2e/models/resnet50_fake_weights.mlir.test"
   "iree/tests/e2e/models/unidirectional_lstm.mlir.test"
+  "iree/tests/e2e/regression/globals.mlir.test"
```

Comment:
Maybe add `TODO(#5716)` to remind us the reason to disable it.

Reply:
That's already there a few lines up.